--- a/src/user.h
+++ b/src/user.h
@@ -17,9 +17,9 @@
 #ifndef __USER_H__
 #define __USER_H__
 
-#define USERS 16
+#define USERS 4
 
-#define OUTPACKETQ_LEN 4		/* Note: 16 users * 1 packet = 1MB */
+#undef OUTPACKETQ_LEN 4		/* Note: 16 users * 1 packet = 1MB */
 /* Undefine to have no queue for packets coming in from tun device, which may
    lead to massive dropping in multi-user situations with high traffic. */
 
